### ISSUE TITLE
?Ajout info livre aide?

### ISSUE THREAD

**Issue by FlynFlykk** [JulienNigon/SpellcasterUniversityPrivateBeta#19]
Je n'ai pas parlé des slots extérieurs
Ni du clic droit pour passer de l'illu au texte de la carte et vis versa

**Comment by JulienNigon** [comment]
Oui, Mathilde a pensé aussi aux aberrations (et aux slots exterieurs). Mais me semble pas dramatique.
